invoices/setup_test.go

```diff
@@ -0,0 +1,11 @@
+package invoices
+
+import (
+	"testing"
+
+	"github.com/lightningnetwork/lnd/kvdb"
+)
+
+func TestMain(m *testing.M) {
+	kvdb.RunTests(m)
```

Comment:
I wonder why we now have the kvdb dependency in the invoice package, I guess just because the helper is defined there and will this change when the sql invoice PR is merged ?

Reply:
Yeah, this is a little "hack" to start and initialize the embedded Postgres instance for the case when we run tests with the PostgreSQL KV driver.